From: Gordon Ball <gordon@chronitis.net>
Date: Tue, 15 Dec 2020 13:24:41 +0000
Subject: Patch out prefix_path test

The test checks for a specific string in the executable path, the
existence of which depends on the layout of the build directory.
---
 test/test_xsystem.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/test_xsystem.cpp b/test/test_xsystem.cpp
index 18f2ab7..adbd47d 100644
--- a/test/test_xsystem.cpp
+++ b/test/test_xsystem.cpp
@@ -26,7 +26,7 @@ namespace xtl
 
         EXPECT_NE(prefix.size(), exec_path.size());
         EXPECT_TRUE(std::equal(prefix.cbegin(), prefix.cend(), exec_path.cbegin()));
-        EXPECT_NE(exec_path.find("test_xtl"), std::string::npos);
+        //EXPECT_NE(exec_path.find("test_xtl"), std::string::npos);
     }
 }
 
